icechunk/src/session.rs

```diff
@@ -256,6 +266,18 @@ impl Session {
         Ok(())
     }
 
+    pub async fn delete_chunks(
+        &mut self,
+        node_path: &Path,
+        coords: Vec<ChunkIndices>,
+    ) -> SessionResult<()> {
+        let node = self.get_array(node_path).await?;
+        for coord in coords {
+            self.set_node_chunk_ref(node.clone(), coord, None).await?
```

Comment:
I don't see how to avoid this clone given that `set_node_chunk_ref` destructures `node`